contentcuration/contentcuration/frontend/shared/views/files/FileStorage.vue

```diff
@@ -98,7 +98,7 @@
         return 'greenSuccess';
       },
       storageRequestUrl() {
-        return window.Urls.storage_settings();
+        return window.Urls.request_storage();
```

Comment:
Good catch! This endpoint is only for posting the request, so it should be `'{}/#/storage'.format(window.Urls.settings())` (not sure if the / in between needs to be there)